src/autopas/selectors/ContainerSelector.h

```diff
@@ -53,6 +53,27 @@ class ContainerSelector {
    */
   void selectContainer(ContainerOption containerOption, ContainerSelectorInfo containerInfo);
 
+  /**
+   * Set new  boundaries, convert the container and reevaluate particle ownership.
+   * @param boxMin
+   * @param boxMax
+   */
+  void resizeBox(const std::array<double, 3> &boxMin, const std::array<double, 3> &boxMax) {
+    _boxMin = boxMin;
+    _boxMax = boxMax;
+
+    // depening on their position convert particles to the corresponding ownership state
+    for (auto &p : *_currentContainer) {
+      if (utils::inBox(p.getR(), _boxMin, _boxMax)) {
+        p.setOwnershipState(OwnershipState::owned);
+      } else {
+        p.setOwnershipState(OwnershipState::halo);
+      }
+    }
```

Comment:
NO!
Previously owned particles should be removed from the container!
And halo particles should probably not be allowed at this stage.

Reply:
done 5a7fab2